src/Doc.js

```diff
@@ -116,7 +116,10 @@ class Doc extends DocBase {
 
     const embed = this.baseEmbed()
     embed.title = 'Search results:'
-    embed.description = searchResults.map(el => `**${el.link}**`).join('\n')
+    embed.description = searchResults.map(el => {
+      const prefix = el.embedPrefix
+      return `${prefix ? `${prefix} ` : ''}**${el.link}**`
```

Comment:
```suggestion
      return `${el.embedPrefix ?? ''}**${el.link}**`
```

Reply:
the reason i used a tenary was to apply a space after the embed prefix if it was present, but not have a leading space if there was no prefix